src/include/txn.h

```diff
@@ -261,11 +273,13 @@ struct __wt_txn {
      *	ids < snap_min are visible,
      *	everything else is visible unless it is in the snapshot.
      */
-    uint64_t snap_min, snap_max;
+    uint64_t snap_max, snap_min;
```

Comment:
better to use the new structure here as well.

Reply:
@bvpvamsikrishna Changing the existing `snap_max`, `snap_min` to use the new structure will have a lot of changes to the existing visibility logic. Because all the existing code that directly dereferences the txn to get the snapshot details like `txn->snap_max` must be changed to `txn->snapshot_details.snap_max`.